src/main/java/com/cburch/logisim/gui/menu/MenuHelp.java

```diff
@@ -155,7 +155,7 @@ public void openProjectWebsite() {
     if (!browserIntegrationSupported()) return;
     try {
       if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
-        Desktop.getDesktop().browse(new URI(Main.APP_URL));
+        Desktop.getDesktop().browse(new URI(Main.ORG_URL));
```

Comment:
Your PR is about VHDL/Verilog header. So why is Help menu or About altered in this PR?